### DISCUSSION TITLE
Terminology clarification request: non-GKI kernel

### DISCUSSION THREAD

**Discussion by theseanl** [tiann/KernelSU#1997] [Q&A] [answered]
It is being said that KernelSU is discontinuing support for "non-GKI kernel", but there doesn't seem to be much information on what this means.

The official installation page https://kernelsu.org/guide/installation.html indicates two methods of installation: GKI and LKM. Does the recent announcement mean that LKM mode installation is no longer being supported? 
How do I know my device "has non-GKI kernel"?

**Comment by tiann** [comment, marked as answer]
1. GKI and LKM are both supported.
2. Non GKI kernel means kernel versions below than 5.10.
